src/components/App.js

```diff
@@ -366,6 +367,16 @@ class App extends React.Component {
     });
   };
 
+  sendCommand = (e) => {
+    e.preventDefault();
+    try {
+      const commandObject = JSON.parse(this.state.bidiCommand);
+      this.#client.sendCommand(commandObject.method, commandObject.params);
+    } catch (e) {
+      console.log({ e });
+    }
```

Comment:
nit: let's use a different error variable name to avoid shadowing the e parameter from the surrounding function.

I would also console.error instead of log. When you submit an invalid message, the only user feedback we have at the moment is the devtools' console panel, so let's make it visible.
```suggestion
    } catch (error) {
      console.error({ error });
    }
```

Reply:
Sure, updated ✅